scripts/schema/loader.py

```diff
@@ -195,6 +195,8 @@ def merge_fields(a, b):
                     asd['reusable']['top_level'] = bsd['reusable']['top_level']
                 else:
                     asd['reusable'].setdefault('top_level', True)
+                if 'order' in bsd['reusable']:
+                    asd['reusable']['order'] = bsd['reusable']['order']
```

Comment:
Yeah, we reuse `process` at `Target.process` and since `hash` is reused in `process` we need a way to either set the order for `hash` to `1` or the order for `process` to `3`